test/e2e/satokensigner_test.go

```diff
@@ -27,17 +27,20 @@ func TestSATokenSignerControllerSyncCerts(t *testing.T) {
 	ctx := context.Background()
 
 	// wait for the operator readiness
+	t.Logf("Waiting for true, false, false")
```

Comment:
t.Log, until 1.14, is not logging until the test finishes but if it panics we don't have logs - I'd do klog.Infof for now (don't forget klog.Init